src/Tribe/Ticket_Object.php

```diff
@@ -547,22 +567,36 @@ public function managing_stock( $manages_stock = null ) {
 		 * @return int
 		 */
 		public function inventory() {
+
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: fd2868c3b3ae3d98ef4d6bf4889e9c78</sub>